PerformanceTest_Static_Pressure_PlantarDorsal_ShinCalf_v2.py

```diff
@@ -0,0 +1,413 @@
+        mfDorsalMaxPressure = float(np.max(avgmfDorsal)*6.895)
+      
+        
+        instepDorsalContact = float(np.count_nonzero(avginstepDorsal)/plantDorDat.dorsalInstepSensNo*100)
+        instepDorsalPressure = float(np.mean(avginstepDorsal)*6.895)
+        instepDorsalMaxPressure = float(np.mean(avginstepDorsal)*6.895)
+
+        
+        plantarContact = float(np.count_nonzero(avgPlantar)/plantDorDat.LplantarSensNo*100)
+        plantarPeakPressure = float(np.max(avgPlantar)*6.895)
+        plantarAvgPressure = float(np.mean(avgPlantar)*6.895)
+        plantarSDPressure = float(np.std(avgPlantar)*6.895)
+        plantarTotalPressure = float(np.sum(avgPlantar)*6.895)
+        
+        
+        toeContact = float(np.count_nonzero(avgToe)/plantDorDat.LplantarToeSensNo*100)
+        toePressure = float(np.mean(avgToe)*6.895) 
+        
+        
+        ffContact = float(np.count_nonzero(avgFF)/plantDorDat.LplantarForefootSensNo*100)
+        ffPressure = float(np.mean(avgFF)*6.895)
+        
+        
+        mfContact = float(np.count_nonzero(avgMF)/plantDorDat.LplantarMidfootSensNo*100)
+        mfPressure = float(np.mean(avgMF)*6.895)
+        
+        heelContact = float(np.count_nonzero(avgHeel)/plantDorDat.LplantarHeelSensNo*100)
+        heelPressure = float(np.mean(avgHeel)*6.895)
+        
+        
+        
+        fd_outcomes = pd.DataFrame([[subject, config, trial, dorsalContact, meanDorsalPressure, maxDorsalPressure,sdDorsalPressure,covDorsalPressure,totalDorsalPressure,
+                       ffDorsalContact, ffDorsalPressure, ffDorsalMaxPressure, mfDorsalContact, mfDorsalPressure,mfDorsalMaxPressure,
+                       instepDorsalContact, instepDorsalPressure,instepDorsalMaxPressure,plantarContact, plantarAvgPressure, plantarPeakPressure,
+                       plantarSDPressure, plantarTotalPressure,toeContact, toePressure, ffContact, ffPressure, mfContact, mfPressure, heelContact, heelPressure]], 
+                        
+                                   columns=['Subject','Config', 'Trial',
+                                'dorsalContact', 'meanDorsalPressure','maxDorsalPressure','sdDorsalPressure','covDorsalPressure','totalDorsalPressure',
+                             
+
+                                'ffDorsalContact', 'ffDorsalPressure', 'ffDorsalMaxPressure', 'mfDorsalContact', 'mfDorsalPressure', 
+                                'mfDorsalMaxPressure', 'instepDorsalContact', 'instepDorsalPressure','instepDorsalMaxPressure',
+                             
+                                'plantarContact', 'meanPlantarPressure', 'maxPlantarPressure', 'sdPlantarPressure', 'totalPlantarPressure',
+                             
+                                'toeContact', 'toePressure', 'ffContact', 'ffPressure',
+                                'mfContact', 'mfPressure', 'heelContact', 'heelPressure'])
+        
+
+        
+    else: 
+        fd_outcomes =  pd.DataFrame([['NA']*20])
```

Comment:
I think the column names should be included here as well. it won't matter if it is not the first file processed if there is no column names, but if it is the first file processed it will not write the column names in the csv that we want